server/routes/index.js

```diff
@@ -0,0 +1,37 @@
+import controller from '../controllers';
+import * as auth from './auth';
+
+const userController = controller.user;
+const docController = controller.document;
+const roleController = controller.role;
+
+export default (app) => {
+  app.get('/info', (req, res) => res.status(200).send({
+    message: 'Welcome to Document Management System!',
+  }));
+
+  app.post('/roles', auth.verifyToken, auth.adminAccess, roleController.createRole);
```

Comment:
For APIs with the same endpoint but different action you can just use app.route('the_common_route').post('controller').get('controller')